tracer/src/Datadog.Trace/Ci/EventModel/TestEvent.cs

```diff
@@ -2,14 +2,14 @@
 // Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
+#nullable enable
 
-namespace Datadog.Trace.Ci.EventModel
+namespace Datadog.Trace.Ci.EventModel;
+
+internal class TestEvent : CIVisibilityEvent<Span>
 {
-    internal class TestEvent : CIVisibilityEvent<Span>
+    public TestEvent(Span span)
+        : base(SpanTypes.Test, 2, span)
```

Comment:
Why is this version incremented to 2 while the newly added types (e.g. `TestModuleEvent`) have a version value of 1?

Reply:
This is part of the event based payload in ci visibility, version 1 of the test event serializes as a normal Span. Version 2 includes Test suite visibility so the backend will search the suite id and module id in the root of the event. Because I’m upgrading all frameworks to test suite visibility I have to send this version 2 to the backend in order to make it work.

Reply:
Ok, and it's expected that new types like `TestModuleEvent` and `TestSessionEvent` have a version 1?

Reply:
yes, is the first time we introduce those events, for that reason is the version 1 for each one.